docs/cookbook/domain/add_command.md

```diff
@@ -0,0 +1,84 @@
+# How create custom command
+
+## Creating domain command
+
+Domain command should represent single transactional business logic.  
+Class that represents is, should implement ```Ergonode\EventSourcing\Infrastructure\DomainCommandInterface```
```

Comment:
dwa razy jest represents, moze po prostu zamiast całej tej linijki "This class should implement"